docs/pages/consent/subscribe-broadcast.mdx

```diff
@@ -0,0 +1,189 @@
+
+Alternatively, you can send an example broadcast message via CLI and cURL instead of using the subscribe-broadcast page. 
+
+In this case, we can send a POST request to the `http://localhost:6989/broadcast` endpoint.
+
+As with the subscribe-broadcast page, the message is sent to the address(es) in the `addresses.json` file at the root of the `broadcast-example-api` directory.
+
+#### Request body
+
+| **Parameter** | **Format** | **Description** |
+| --- | --- | --- |
+| text | string | **Required**. Message text you want to broadcast. |
+| address | string | **Required**. The broadcasting raw Ethereum 0x wallet address. |
+
+#### Example request
+
+```html
+curl -X POST http://localhost:6989/broadcast \
+-H "Content-Type: application/json" \
+-d '{
+    "text": "Hello, this is a broadcast message!",
+    "address": "<Replace with your broadcast wallet address>"
+}'
+```
+
+#### Response body
+
+| **Parameter** | **Format** | **Description** |
+| --- | --- | --- |
+| broadcastId | string | Unique ID for the broadcast. You can use it to track or query for the broadcast and its status. |
+
+```html
+{"broadcastId":"ee21a960-310e-446b-ac41-5665ae1906e6"}
+```
+
+## Check the status of the example broadcast message
+
+In addition to checking the main inbox for the subscribed address, you can use the example subscribe-broadcast page to check on the status of the example broadcast message.
+
+Go to `http://localhost:6989/broadcast?broadcastId=<broadcastId>` 
+
+For example, [http://localhost:6989/broadcast?broadcastId=0624abaf-0312-4653-8236-1ff6bda21328](http://localhost:6989/broadcast?broadcastId=0624abaf-0312-4653-8236-1ff6bda21328)
+
+The page displays the following information about the broadcast:
+
+```json
+{
+  "id": "c49ce40e-93ea-489c-938f-89d46b7cc6e9",
+  "address": "0x62EEd858af7590fbCaE803d208c53ddBb0D1309c",
+  "message": "here's a new test message yo",
+  "recipients": 1,
+  "sent": 1,
+  "startTime": "2024-08-23T00:09:02.408Z",
+  "status": "completed",
+  "endTime": "2024-08-23T00:09:02.655Z"
+}
+```
+
+| Parameter | Format | Description |
+| --- | --- | --- |
+| id | string | ID of the broadcast |
+| address | string | Address of the broadcast sender |
+| message | string | Message sent in the broadcast |
+| recipients | string | Number of recipient addresses included in the broadcast |
+| sent | number | Number of messages sent |
+| startTime | string | Time the broadcast started |
+| status | string | Possible values: `Sending`, `Waiting`, `Completed`, and `Failed`.  |
+| endTime | string | Time the broadcast ended |
+
+### Check the status of the example broadcast message via CLI
+
+Alternatively, you can check the status of the example broadcast message via CLI and cURL instead of using the subscribe-broadcast page. 
+
+In this case, we can send a GET request to the `http://localhost:6989/broadcast` endpoint.
+
+```html
+curl -X GET "http://localhost:6989/broadcast?broadcastId=0624abaf-0312-4653-8236-1ff6bda21328"
+```
```

Comment:
How do https://www.npmjs.com/package/@xmtp/broadcast-sdk and https://github.com/alexrisch/broadcaster-app come into the picture? Is there an opportunity to surface them in this README?